### ISSUE TITLE
NTR: Bladder outlet obstruction

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4906]
**Preferred term label:**
Bladder outlet obstruction

**Synonyms**
obstruction of bladder outlet

**Definition (free text, please give PubMed ID)**
A compression or resistance upon the bladder outflow channel at any location from the bladder neck to urethral meatus, which usually causes lower urinary tract symptoms (LUTS). PMID:16986027

**Parent term (use hpo.jax.org/app)**
HP:0000009 Functional abnormality of the bladder
